<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="libraryLayout">
        <item>
         <widget class="QLabel" name="libraryLabel">
          <property name="text">
           <string>Your library:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="libraryListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="mainContent">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="welcomeText">
         <widget class="QLabel" name="mainTextLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>391</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="itemData">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="detailsLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Item details:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="separator">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="dataFormLayout">
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>23</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="isbnLabel">
                <property name="text">
                 <string>ISBN:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="isbnData"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="titleLabel">
                <property name="text">
                 <string>Title:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="titleData"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="publisherLabel">
                <property name="text">
                 <string>Publisher:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="publisherData"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="yearLabel">
                <property name="text">
                 <string>Year:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="yearData"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="ratingLabel">
                <property name="text">
                 <string>Rating:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="ratingData"/>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>General informations</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="editItemButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removeItemButtonButton">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBook">
    <property name="title">
     <string>Book</string>
    </property>
    <addaction name="addItemAction"/>
    <addaction name="removeItemAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuBook"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addItemAction"/>
   <addaction name="removeItemAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addItemAction">
   <property name="text">
    <string>Add Item</string>
   </property>
   <property name="toolTip">
    <string>Add a library item</string>
   </property>
  </action>
  <action name="removeItemAction">
   <property name="text">
    <string>Remove Item</string>
   </property>
   <property name="toolTip">
    <string>Remove a library item</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About BookApp</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
